components/Grid/grid.css

```diff
@@ -0,0 +1,60 @@
+.grids-wrapper {
+  align-items: center;
+  align-self: center;
+  background-color: var(--white);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-self: center;
+  margin: 2rem;
+  min-width: fit-content;
+  padding: 2rem;
+  width: 80%;
+}
+.grid {
+  display: grid;
+  gap: 1rem;
+  width: 100%;
+}
+.grid-cols-2 {
+  grid-template-columns: repeat(2, 1fr);
+}
+.grid-cols-3 {
+  grid-template-columns: repeat(3, 1fr);
+}
+.grid-holy-grail {
+  grid-template-areas:
+    "header header header"
+    "nav main side"
+    "footer footer footer";
+  grid-template-columns: 200px 1fr 200px;
+  grid-template-rows: auto 1fr auto;
+}
+.col-1,
+.col-2,
+.col-3,
+.grid-holy-grail header,
+.grid-holy-grail nav,
+.grid-holy-grail main,
+.grid-holy-grail aside,
+.grid-holy-grail footer {
+  background-color: var(--light-purple);
```

Comment:
you could rewrite this as .grid-holy-grail>* { ....css properties }

Reply:
That does make it very easier and optimised. Learnt something new today. Thank you for the suggestion.